js/controller.js

```diff
@@ -97,6 +97,7 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
       "isSkipAdClicked": false,
       "isInitialPlay": false,
       "isFullScreenSupported": false,
+      "isVideoFullScreenSupported": false,
```

Comment:
Let's rename isVideoFullScreenSupported to isIOSVideoFullScreenSupported for more clarity